recipes/approvaltests.cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <cstdlib>
+#include <iostream>
+#include "ApprovalTests.hpp"
+
+
+int main(void) {
+    std::cout << ApprovalTests::StringMaker::toString(42) << " is the answer."<< std::endl;
```

Comment:
this is risky. We would better use a symbol from ApprovalTests library, or the linker will discard the object file, and we have no chance of detecting missing requirements/system requirements.